content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
+We hope you will forgive us (*send everyone who contributed a cake if you do, please!*) after trying it!
+
+# Key Features
+
+## Let your content shine like never before
+
+If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:
+
+#### Jellyfin 10.4.3
+
+<img src="/images/10.5-release/old.png" name="Old item details page" />
+
+#### Jellyfin 10.5
+
+<img src="/images/10.5-release/new.png" name="New item details page" />
+
+*With 10.5, we wanted to reimagine Jellyfin's Web interface*
+
+Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was the jungle when
+I took that screenshot of the *Jungle's Book* wallpaper. Do you feel it like me?
+
+Notice also that the header of the UI has been reworked a little bit in a *slightly* lighter color (instead of the plain color used by the theme)
+
+Shoutout to all the Web team for their invaluable contributions in mking this happen, specially to *[Julien](https://github.com/mrtimscampi)* and 
+*[grafixeyehero](https://github.com/grafixeyehero)*.
+
+You can check the redesign in our [demo server](https://demo.jellyfin.org/stable)
+
+## Faster than the sound and cleaner than a surgery room
+
+As you already know, we're still a really young community that want to provide the best self-hosted media server out there. As with every release,
+**we tried to clean up and optimize the old code** that we inherited when we forked Emby's code. And that's what we have been doing since
+the day we released 10.4.3. A lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.
+
+Let's start with the frontend, as it's what you see everyday in the end. A lot of work has been done to simplify the code and ease the transition
+to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. While
+all the benefits of this are not visible for you in the daily use just yet, it's a necessary step towards a better application. In fact, it already paid off in application speed. See this video:
+
+<video autoplay muted loop preload="none" id="preview">
+    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
+</video>
+
+Not only you're seeing the speed differences, you're seeing the new animations! Images **will fade in** every time you move to another page. You **can** disable that in settings if you want to come back the old behaviour though.
+
+The old-standing issue that many of you had (by the way, introduced by the *fellow writer* of this post 🙄) caused by the images has been fixed in this version
```

Comment:
Which issue are you referring to ? The reader might not know, they might be a new user looking at what the latest release brings, etc.